src/lib/core/components/computations/StartPage.tsx

```diff
@@ -67,120 +68,117 @@ export function StartPage(props: Props) {
                     {app.description}
                   </span>
                 </div>
-                {/* <Link
-                to={`${baseUrl}/new/${app.name}`}
-                style={{
-                  pointerEvents: plugins[app.name] ? 'auto' : 'none',
-                }}
-              >
-                <div style={{ fontWeight: 'bold', fontSize: '1.2em' }}>
-                  {app.displayName}
-                  {plugins[app.name] ? '' : <i> (Coming soon!)</i>}
-                </div>
-                <div>{app.description}</div>
-              </Link> */}
                 {app.visualizations?.map((vizType, index) => {
                   const disabled = plugins[app.name] === undefined;
                   const VizSelector = plugins[app.name]
                     ? plugins[app.name].visualizationTypes[vizType.name]
                         .selectorComponent
                     : helperPlugin.visualizationTypes['scatterplot']
                         .selectorComponent;
-
+                  const plugin = plugins[app.name];
```

Comment:
Need to assert `plugin` is not undefined

Reply:
Apologies, that chunk of code changed with [this PR](https://github.com/VEuPathDB/web-eda/pull/1213) and I forgot to push the merged code.